sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2021-07-16)
 
-### Key Bug Fixes
-- Fix exceptions when use in Static Web Apps.
+### Bugs Fixed
+- Fixed exceptions when the library is used in Static Web Apps.
```

Comment:
### Features
- Add `WebPubSubRequest` input binding to support Static Web Apps.